SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoft.cshtml

```diff
@@ -0,0 +1,67 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+
+@model SeaPublicWebsite.Models.EnergyEfficiency.AccessibleLoftViewModel
+@{
+    ViewBag.Title = "Accessible loft";
+}
+
+@section BeforeMain {
+    @await Html.GovUkBackLink(new BackLinkViewModel
+    {
+        Text = "Back",
+        Href = Model.BackLink,
+        
+        // Note: some of the "Back" links in this journey can't use the default browser behaviour
+        //  so we should control all of them explicitly
+        OverrideWithJavascript = false
+    })
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <form action="@Url.Action("AccessibleLoft_Post", "EnergyEfficiency", new {reference = Model.Reference})" method="post" novalidate>
+            @(Html.HiddenFor(m => m.EntryPoint))
+            @Html.AntiForgeryToken()
+            
+            @{
+                Func<object, object> details = 
+                    @<text>
+                        <p class="govuk-body">
+                            You may have a loft hatch, access ladder or stairs that could allow an insulation installer to get into the loft space.
+                        </p>
+                     </text>;
+            }
+            
+            @(await Html.GovUkRadiosFor(m => m.AccessibleLoft,
+                fieldsetOptions: new FieldsetViewModel
+                {
+                    Legend = new LegendViewModel
+                    {
+                        Text = "Is there access to your loft?",
+                        Classes = "govuk-fieldset__legend--l",
+                        IsPageHeading = true
+                    }
+                },
+                new HintViewModel
```

Comment:
The design has an EPC hint, should that be included on this ticket?

Reply:
There is no information on the EPC about loft space, just about roof insulation, so there's no hint to be added.
On the topic of the ticket design, It was created before having the discussion about the 'skip link'. (Plus, none other roof questions have a skip link). I converted the skip link into the not sure option, and depending on how we want to proceed further change it in the future.